tests/system/robot/chromeTests.py

```diff
@@ -159,6 +159,10 @@ def test_mark_aria_details_role():
 				<div id="footnote-details" role="doc-footnote">details with role doc-footnote</div>
 				<div id="comment-details" role="comment">details with role comment</div>
 				<div id="definition-details-as-role" role="definition">details with role definition</div>
+				<p>
+					<span role="term">definition</span>:
+					<span id="definition-details-as-role" role="definition">details with role definition</span>
+				</p>

```

Comment:
These shouldn't both exist (two elements with `id="definition-details-as-role"`.
I'd also like the `<dfn>` example added. It seems both the `role="term"` / `role="definition"` case and the `<p> A description for a <dfn>term</dfn> is in this paragraph.</p>` case share the similar problem about `aria-details` pointing to the **term** rather than the **definition**. Illustrating this in the tests will be helpful when having the discussion later.

Reply:
I've added documentation to the test samples and fixed it so we consistently point to the term in cfb74bf